cmd/sst-ctl/main.go

```diff
@@ -84,3 +88,44 @@ func subCmdInfo(args []string) error {
 	fmt.Println(utils.DumpJSON(info))
 	return nil
 }
+
+func subCmdBF(args []string) error {
+	flags := flag.NewFlagSet("bf", flag.ExitOnError)
+	flags.BoolVar(&enable, "enable", false, "enable feature")
+	flags.BoolVar(&disable, "disable", false, "disable feature")
+	addGlobalFlags(flags)
+
+	if err := flags.Parse(args); err != nil {
+		return err
+	}
+
+	if !enable && !disable {
```

Comment:
About arguments validation: we check/complain for no flags, but what if both are present? Then the result is non-obvious i.e. disable is always set (regardless of order)

Reply:
Fixed